package/MDAnalysis/core/AtomGroup.py

```diff
@@ -5028,6 +5037,46 @@ def _matches_unpickling(self, anchor_name, n_atoms, fname, trajname):
         else:
             return False
 
+    def transfer_to_memory(self, frame_interval=1):
+        """Replace the current trajectory reader object with one of type
+        :class:`MDAnalysis.coordinates.memory.MemoryReader` to support in-place
+        editing of coordinates.
+
+        :Arguments:
+             *frame_interval*
+                 Read in every nth frame.
+
+        .. versionadded:: 0.15.0
+        """
+
+        from ..coordinates.memory import MemoryReader
+
+        if self.trajectory.format != "array":
```

Comment:
Are you using "memory" or "array" as the format name? `MemoryReader.format == "memory" ?